index.js

```diff
@@ -72,7 +72,7 @@ function fastifyCors (fastify, opts, next) {
   // remove most headers (such as the Authentication header).
   //
   // This route simply enables fastify to accept preflight requests.
-  fastify.options('*', { schema: { hide: hideOptionsRoute } }, (req, reply) => {
+  fastify.options('/*', { schema: { hide: hideOptionsRoute } }, (req, reply) => {
```

Comment:
why?

Reply:
when we register the url into find-my-way, we do:

```
prefix + url
```

so, in this case:

```
path param: "/:id" + "*" = "/:id*"
static prefix: "/foo" + "*" = "/foo*"
```

while the 2nd seems to work (but it can generate conflicts in any case), the first one does not work because it is a path parameter + a whilcard